servers/media-server/src/server/connector.rs

```diff
@@ -33,7 +32,7 @@ use super::MediaServerContext;
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 pub struct ConnectorArgs {
-    /// Message Queue URI in the form of `amqp://user:pass@host:port/vhost`
+    /// Message Queue URI in the form of `http://host:port` or `nats://host:port`
```

Comment:
maybe we should setting http as default, nats is only for complex system. I guest that most of user will use http instead

Reply:
but what will be the default value, just a arbitrary localhost URL will a random common port?